### ISSUE TITLE
[Badger 2040] Strange lines and ghost on e-paper

### ISSUE THREAD

**Issue by niutech** [pimoroni/pimoroni-pico#591]
I left my Badger 2040 laying disconnected overnight with blank screen and this morning I saw strange dark lines (strokes), a border and a white ghost of an image I displayed yesterday, which should be cleared. What is causing these lines and is it normal?

![Badger 2040 lines](https://user-images.githubusercontent.com/384997/205267423-40b9535b-fb2d-41f5-bc60-48925333b9b0.jpg)

**Comment by Gadgetoid** [comment]
Is that the Chrome dinosaur?

In my experience e-paper is prone to weirdness like this. It pulls (or flips) ink particles in a substrate and they're not very well-behaved. This is odd, but not alarming. We might be pushing things a little far on TURBO :grimacing:

**Comment by niutech** [comment]
Yes, this is the ghost from my [Dino Game](https://github.com/niutech/dino-badger2040).
